resources/js/components/basic-select-options.tsx

```diff
@@ -32,6 +32,8 @@ export default class RankingSelectOptions extends React.PureComponent<Props> {
 
   private href(id: number | null) {
```

Comment:
At this point the component should probably just accept a callback to get the `href`.
Also, `modifiers` passed to `SelectOptions` should no longer be `spotlight`.

Reply:
well, multiplayer one is still being casted from blade via registerComponents thing so not sure how to go with callback thing considering that :thinking: